package.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "skyway",
+  "name": "skyway-js",
   "version": "1.0.1",
-  "description": "The JS client library for SkyWay",
+  "description": "The official JavaScript SDK for SkyWay",
   "main": "dist/skyway.js",
-  "module": "src/peer.js",
```

Comment:
Did this line cause any problems?

See more about what this line does -> https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for
But I don't think it reachs de facto standard yet. So it is no problem to remove.

Reply:
no I don't think it caused any problems. it just seemed unnecessary so I removed it. I'll add it back in (I'll rebase it to clean up the commit log before it get's merged)